ShitFood.Api/GetShit.cs

```diff
@@ -9,33 +9,76 @@
 using Newtonsoft.Json;
 using System.Net.Http;
 using ShitFood.Api.FoodHygieneApi;
+using GoogleApi;
+using GoogleApi.Entities.Places.Search.Find.Request;
+using GoogleApi.Entities.Places.Search.Find.Response;
+using System.Linq;
+using System.Collections.Generic;
 
 namespace ShitFood.Api
 {
     public static class GetShit
     {
+        private static string _googleApiKey = Environment.GetEnvironmentVariable("GoogleApiKeyFromKeyVault");
+
         [FunctionName("GetShit")]
         public static async Task<IActionResult> Run(
             [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = null)] HttpRequest req, ILogger log)
         {
             string lat = req.Query["lat"];
             string lon = req.Query["lon"];
 
-            var establishments = await GetBadFoodHygieneRatings(lat, lon);
```

Comment:
Traditionally, especially with Google, you tend to see `lat, lng` rather than `lat, lon`

Reply:
I can change the name of the parameter you provide to that, I think it's nicer too